setup.py

```diff
@@ -32,6 +32,13 @@ def get_and_set_version():
     packages=['torchacc'] + ['torchacc.' + \
                              pkg for pkg in find_packages('torchacc')],
 
+    # add console_scripts
+    entry_points={
+        'console_scripts': [
+            'consolidate_and_reshape_ckpts = torchacc.utils.consolidate_and_reshard_ckpts:main',
```

Comment:
Rename `consolidate_and_reshape_ckpts` to `consolidate_and_reshape_fsdp_ckpts` (Add fsdp information to the file name).